forest-node/ansible/roles/forest/tasks/main.yml

```diff
@@ -0,0 +1,62 @@
+---
+# tasks file for role/forest
+- name: Upgrade all apt packages
+  apt: upgrade=dist force_apt_get=yes
+
+- name: Create a login user
+  user:
+    name: "{{ user }}"
+    state: present
+
+- name: Add public key to authorized_keys
+  authorized_key:
+    user: "{{ user }}"
+    state: present
+    key: "{{ lookup('file', '~/.ssh/id_rsa.pub') }}"    # assuming the name of the public key file is `id_rsa.pub`
```

Comment:
I don't have my public key at `~/.ssh/id_rsa.pub`. How about managing the keys through Digital Ocean?

Reply:
You just have to change the directory to where your public key exists.

Reply:
Why can't we either put the key in a configuration file or (better yet) just manage the keys through Digital Ocean?

Reply:
But we already created a set of authorized keys when we created the droplet, right? Can't we reuse those authorized keys?

Reply:
Yes, we are using the same keys but in this case, we are adding it for the new user to log in via its own authorized-keys folder. While root has its own, the new user needs its own as well

Reply:
Can't we copy the authorized keys from the root user?

Reply:
Erm, well, this is a possibility (I have an imagination of the playbook in my head already) but can I understand what the issue is here exactly? Is there a problem or disadvantage with copying the keys from the local machine 

Meanwhile, I'd cook up a play now and try if that works as well.

Reply:
The only problem is that `~/.ssh/id_rsa.pub` might not be where users keep their public key. And we're already specifying the public key using a fingerprint. It would simplify the process if we only used the fingerprint.